src/api/app/models/package.rb

```diff
@@ -878,8 +878,9 @@ def build_result_old(repository, view = [])
     Buildresult.find(project: project, package: self, repository: repository, view: view)
   end
 
-  def buildresults
-    results = Buildresult.find_hashed(project: project.name, package: name, view: 'status', multibuild: '1', locallink: '1')
+  def buildresults(prj = nil)
+    prj = project unless prj
```

Comment:
There are spaces around the = operator. My point was rather that if the default value for prj is self.project, then we could define that the method head already.